lib/src/models/group_item.dart

```diff
@@ -0,0 +1,26 @@
+part of lunch_picker;
+
+class GroupItem {
+  String name;
+  Group group;
+  int lastPicked;
+  bool hasPicked;
+
+  GroupItem(this.name, this.group, {this.lastPicked: 0, this.hasPicked: false});
+
+  GroupItem.fromJson(Map json) {
+    this.name = json['name'];
+    this.group = new Group.fromJson(json['group']);
```

Comment:
For now, yep. If / when group is extended to contain more info than just the name, we can update this.